packages/application-shell/src/components/application-shell-authenticated/application-shell-authenticated.tsx

```diff
@@ -202,6 +202,7 @@ export const ApplicationShellAuthenticated = (
                     defaultFlags={props.defaultFeatureFlags}
                   >
                     <>
+                      <div id="above-top-navigation" />
```

Comment:
Nit: putting it here, will "push the page down" thus creating a scrollbar, which might not be necessary.

I think we can try including it into the global notifications container, as a sibling of the global notifications list.

```diff
+<div id="above-top-navigation" />
<NotificationsList domain={DOMAINS.GLOBAL} />
```

This has the benefit of using the existing grid layout so there should be no potential "issues" with the layout

<img width="1349" alt="image" src="https://user-images.githubusercontent.com/1110551/236769154-6545817e-be8c-4621-9b7c-67e2082917d0.png">

What do you think?

Reply:
This is much better. No unnecessary scrolling. Just a few minor changes required which can be taken care in MC later. Thanks.